packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1421,6 +1427,74 @@ class FormattedJson extends StatelessWidget {
   }
 }
 
+class JsonViewer extends StatefulWidget {
+  const JsonViewer({
+    required this.json,
+  });
+
+  final String json;
```

Comment:
maybe rename to 'encodedJson'? when I see a var named `json` I immediately think it is a Map

Reply:
Done.